ci/management/commands/generate_reports.py

```diff
@@ -303,3 +352,68 @@ def handle_health_worker_registrations(
                 'Cadre': operator_details.get('role'),
                 'Number of Registrations': count,
             })
+
+    def handle_enrollments(self, sheet, sbm_client, ids_client, start_date,
+                           end_date):
+
+        sheet.set_header([
+            'Message set',
+            'Roleplayer',
+            'Currently enrolled (snapshot at point y)',
+            'Cumulatively enrolled (in last period x)',
+            'Of the cumulatively enrolled (in last period x), number which '
+            'opted out',
+            'Of the cumulatively enrolled (in last period x), the number which'
+            ' completed messages',
+        ])
+
+        subscriptions = self.get_subscriptions(
+            sbm_client,
+            created_before=end_date.isoformat())
+
+        data = {}
+        for idx, subscription in enumerate(subscriptions):
```

Comment:
I don't think that we need to `idx` and `enumerate` here, we don't seem to be using it?